src/Loggers/OwnSplitChannel.cpp

```diff
@@ -107,7 +107,7 @@ void OwnSplitChannel::logSplit(const Poco::Message & msg)
         columns[i++]->insert(msg.getSource());
         columns[i++]->insert(msg.getText());
 
-        [[maybe_unused]] bool push_result = logs_queue->emplace(std::move(columns));
+        logs_queue->emplace(std::move(columns));
     }
 
     /// Also log to system.text_log table, if message is not too noisy
```

Comment:
It solves the issue with `InternalTextLogsQueue::queue::queue_mutex`, but this thread will write message into `system.text_log` and lock other mutexes (e.g. `text_log_mutex` and `SystemLogBase::mutex`), so seems like similar lock-order-inversions are still possible if `text_log` try to allocate memory and get into `OvercommitTracker`